### DISCUSSION TITLE
How to test the qt6 version

### DISCUSSION THREAD

**Discussion by bodograumann** [qutebrowser/qutebrowser#7133] [Q&A] [answered]
I don’t want to further generate noise in the https://github.com/qutebrowser/qutebrowser/issues/5395 issue, so let’s continue here. 

Trying to build on the `qt6-v2` branch seems to still include qt5-webengine and when I run it, I get
> 14:54:16 ERROR    webview    tabbedbrowser:show_error_page:968 Renderer process crashed (status 159)

I probably need to install some additional qt6-* libraries on the system⁉ This is on arch, so for the normal qutebrowser I can just use pacman.

**Comment by The-Compiler** [comment, marked as answer]
Can you show the `--version` output you get now? You will indeed need `python-pyqt6` and `python-pyqt6-webengine`. Maybe more, but I think at that point qutebrowser will pick it up and suggest what's missing.

I'd also suggest `mkvenv.py` usually for playing around, but just realized that's actually not quite ready yet.

**Reply by bodograumann** [reply]
I did build with `mkvenv.py`. So if it’s not ready for qt6 that explains why I am still on qt5:
```
qutebrowser v2.5.0
Git commit: f3adcff53 on qt6-v2 (2022-04-20 20:11:09 +0200)
Backend: QtWebEngine 5.15.2, based on Chromium 83.0.4103.122
Qt: 5.15.2
```

**Reply by bodograumann** [reply]
I have installed `python-pyqt6-webengine` system-wide and also tried `python scripts/mkvenv.py --pyqt-type link`, but could not get the result to use qt6.
What do I need to do?

**Reply by The-Compiler** [reply]
If you install dependencies system-wide (I assume you already have the other dependencies if you have qutebrowser installed via pacman), you can just run `python3 -m qutebrowser --temp-basedir` in the source tree, without needing `mkvenv.py`. Probably going to fix that script early next week, though.

**Reply by bodograumann** [reply]
Ok that works. Just cant be in the virtual environment :-D

```
qutebrowser v2.5.0
Git commit: de99a5777 on qt6-v2 (2022-04-22 12:07:41 +0200)
Backend: QtWebEngine 6.3, based on Chromium 94.0.4606.126
Qt: 6.3.0

CPython: 3.10.4
PyQt: 6.3.0
```
